B 280

talmazov opened the pull request:
Code review
-trim tool
- model ops: base, hollow
-surveying

Hollow does not work yet

talmazov committed:
splint implementation

- implemented splint capabilities via weight paint
- after blockout, the remesh op is used now
- UI additions and enhancements

talmazov committed:
implant UI, moved model op panel, fixed surveyor

talmazov committed:
new features, updates

- updated clean mesh op, improved hole filling and edge smoothing
- introduced color blockout in addition to silouette, now when user has "pre-surveyed" the last stored view projection is remember, then when solid blockout op is used that last view is used so the model can be inspected without having to return to original view position

talmazov committed:
trim tool

- operators
- panel

patmo141 committed:
Merge branch 'b_280' into b_280

patmo141 commented:
No idea if this merge worked.  It was super hard to tell what GIT tried to put where

talmazov committed:
hollow op, frame cutter

- incorporated Issam's bmesh hollow op, added perspective cutback
order of HOLLOW op: clean model -> solid base -> remesh -> hollow -> cutback

- incorporated Issam's frame cutter as is, UI workflow improvements

talmazov committed:
authors, version

- included team authors
- moved to v. 1.1

talmazov committed:
Update .gitignore

talmazov committed:
ODC Teeth

- removed layer manager in ODCutils, new one will be implemented for b2.8 as needed
- converted code to b2.8

issamdakir committed:
ODC update

Hirarchy of the addon folders simplified
Registration simplified
Some code errors fixed
Added a control button, new operators to Models operations section and optimized code performance

issamdakir committed:
ODC_fix_06_28_2020_01

solid base and hollow base height fixed
hiden child object when applying link operator fixed
added decimate property
fixed layout buttons locations
changed popup box message icon to orange when prompting user to select an object
fixing message box conflict
TODO : fix preview option on blockout model operator

issamdakir committed:
Update model_ops.py

issamdakir committed:
Update model_ops.py

talmazov committed:
Merge pull request #2 from issamdakir/b_282

ODC_fix_06_28_2020_01

talmazov committed:
Update README.md

talmazov committed:
Merge branch 'master' into b_280

talmazov committed:
moved

conflict from master branch merge
